Automattic/wp-calypso
# Site Transfers: Fix color for pending domain transfer screen

Closes https://github.com/Automattic/dotcom-forge/issues/2600

## Proposed Changes

* This PR changes the text on the pending domain transfer screen for site transfer to `color: var( --studio-gray-70 )` to be consistent with [Figma design](1U910ArebC7gbLF1RtxUxE-fi-124_5938)

### Before

<img width="800" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/2eb9896e-c2b3-42e3-85fb-5ced478c8fcb">

### After

<img width="830" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/f4d13a52-4393-4613-8030-f6315485a0c4">


## Testing Instructions

* Create a simple site
* Navigate to `Settings > General > Transfer` your site (you can use the Calypso live link)
* Negate this [this line](https://github.com/Automattic/wp-calypso/pull/77505/files#diff-fb48d30f501ae2da3a2bbd4f2b12ce7c1780f8c0cbb5394be797fba892775771R26) to trigger the pending domain screen
* Ensure that you see a component like the one below
* Check that the color of the text "There are pending domain transfers..." is set to `color: var( --studio-gray-70 )` and is the same as the `Site Transfer` color:

<img width="837" alt="Screenshot 2023-05-31 at 11 07 22 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/d185b038-4bb3-424e-9aed-57cdddc02a19">

## Pre-merge Checklist
